gulpfile.js

```diff
@@ -50,6 +50,7 @@ var babelOpts = {
     [babelPluginModules, { map: moduleMap }],
   ],
 };
+babelOpts.plugins[0][1].map.stream = 'stream';
```

Comment:
I believe this line is needed to get require to look for a package rather than a module that uses providesModule.